src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -176,10 +176,66 @@ public void initialize(URL location, ResourceBundle resources) {
         newMessageIndicator.visibleProperty().bind(newMessageIndicator.textProperty().isNotEqualTo("0"));
     }
 
+    @FXML
+    BorderPane quotaBlock;
+    @FXML
+    Label quota;
+    @FXML
+    Label provider;
+    @FXML
+    Label quotaDescription;
+    @Inject
+    BoxClient boxClient;
+
+    private void fillQuotaInformation() {
+        try {
+            quotaState = boxClient.getQuotaState();
+        } catch (IOException | QblInvalidCredentials e) {
+            alert(e);
+        }
+        if (quotaState == null) {
+            quotaBlock.setVisible(false);
+            quotaDescription.setVisible(false);
+            return;
+        }
+
+        int ratio = ratioByDiff(quotaState.getSize(), quotaState.getQuota());
+        String quotaDescriptionText = quotaDescription(quotaState.getSize(), quotaState.getQuota());
+        quota.setText(ratio + "%");
+        provider.setMinWidth(ratio);
+        quotaDescription.setText(quotaDescriptionText);
+    }
+
+    int ratioByDiff(long usedQuota, long availableQuota) {
+        return (int) (usedQuota / (double) availableQuota * 100);
+    }
+
+    String quotaDescription(long usedQuota, long availableQuota) {
+        long diff = availableQuota - usedQuota;
```

Comment:
`free` seems to fit better than diff. This method is `quotaDescription` and not `concat` or `sum` ... even if the result is some kind of a sum of strings.